sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/QuestionAnsweringClientOptions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.Language.QuestionAnswering
+{
+    /// <summary> Client options for QuestionAnsweringClient. </summary>
```

Comment:
```suggestion
    /// <summary> Client options for the <see cref="QuestionAnsweringClient"/>. </summary>
```

Reply:
I may not actually be able to change this. This was copied from generated code and I forget why off hand, but if/when it's generated again I won't be able to change it. I can file a bug on the generator, though.

Reply:
I changed it for now, at least. IIRC, once I add authoring support back in I may lose that ability, but I'll address is then if I can.